ddtrace/internal/ci_visibility/git_client.py

```diff
@@ -90,6 +90,9 @@ def _run_protocol(cls, serializer, requests_mode, base_url, _tags={}, _response=
 
         latest_commits = cls._get_latest_commits(cwd=cwd)
         backend_commits = cls._search_commits(requests_mode, base_url, repo_url, latest_commits, serializer, _response)
+        if backend_commits is None:
```

Comment:
If this is `True`, should we not still upload packfiles? The worst case scenario is that we accidentally send info about commits that our backend already knows about, right? Is that problematic?